lib/ansible/module_utils/network/netscaler/netscaler.py

```diff
@@ -248,8 +279,20 @@ def get_nitro_client(module):
         type='bool',
         default=True,
     ),
-)
 
+    mas_proxy_call=dict(
+        default=False,
+        type='bool'
+    ),
+    nitro_auth_token=dict(
```

Comment:
I wonder if in the argspec in the modules you'd want
```python
mutually_exclusive=(['nitro_pass', 'nitro_auth_token'],)
required_together=(['nitro_user', 'nitro_pass'],),
required_one_of=[['nitro_pass', 'nitro_auth_token']],

```

See https://github.com/gundalow/ansible/blob/docs-argspec/docs/docsite/rst/dev_guide/module_utils/basic.rst